pkg/dashboard/objects/releases_test.go

```diff
@@ -0,0 +1,80 @@
+package objects
+
+import (
+	"sync"
+	"testing"
+
+	"gotest.tools/v3/assert"
+	"helm.sh/helm/v3/pkg/action"
+	kubefake "helm.sh/helm/v3/pkg/kube/fake"
+	"helm.sh/helm/v3/pkg/release"
+	"helm.sh/helm/v3/pkg/storage"
+	"helm.sh/helm/v3/pkg/storage/driver"
+)
+
+var (
+	fakeKubeClient *kubefake.PrintingKubeClient
+	fakeStorage    *storage.Storage
+)
+
+func fakeHelmNSConfigGetter(ns string) (*action.Configuration, error) {
+	return &action.Configuration{
+		KubeClient: fakeKubeClient,
+		Releases:   fakeStorage,
+	}, nil
+}
+
+func TestListReleases(t *testing.T) {
+	fakeStorage = storage.Init(driver.NewMemory())
+	err := fakeStorage.Create(&release.Release{
+		Name: "release1",
+		Info: &release.Info{
+			Status: release.StatusDeployed,
+		},
+	})
+	assert.NilError(t, err)
+	err = fakeStorage.Create(&release.Release{
+		Name: "release2",
+		Info: &release.Info{
+			Status: release.StatusDeployed,
+		},
+	})
+	assert.NilError(t, err)
+	err = fakeStorage.Create(&release.Release{
+		Name: "release3",
+		Info: &release.Info{
+			Status: release.StatusDeployed,
+		},
+	})
+	assert.NilError(t, err)
+	err = fakeStorage.Create(&release.Release{
+		Name: "release4",
+		Info: &release.Info{
+			Status: release.StatusDeployed,
+		},
+	})
+	assert.NilError(t, err)
+	err = fakeStorage.Create(&release.Release{
+		Name: "release5",
+		Info: &release.Info{
+			Status: release.StatusDeployed,
+		},
+	})
+	assert.NilError(t, err)
+
+	releases := &Releases{
+		Namespaces: []string{"testNamespace"},
+		HelmConfig: fakeHelmNSConfigGetter,
+		mx:         sync.Mutex{},
```

Comment:
No need to initialize it, since it's not a reference. It would be initialized with default value automatically

Reply:
Understood, Thanks.